rpc/README.md

```diff
@@ -383,13 +383,22 @@ http://localhost:8114
 
 ### `get_cells_by_lock_hash`
 
-Returns the information about cells collection by the hash of lock script.
+Returns the information about live cells collection by the hash of lock script.
 
 #### Parameters
 
     lock_hash - Cell lock script hash
     from - Start block number
     to - End block number
+#### Returns
+
+    block_hash - Refer to block
+    capacity - Cell capacity
+    cellbase_maturity - Cellbase maturity, optional boolean, null means non-cellbse
```

Comment:
RPC should return some determined result, instead of a value which varies overtime.

Reply:
how about change `cellbase_maturity` to enum: none / yes / no ?

Reply:
I prefer returning the epoch with fraction where the cell is created.